themes/_core/js/product.js

```diff
@@ -111,9 +111,10 @@ function updateProduct(event, eventType, updateUrl) {
     }
 
     // New request only if new value
-    if (event !== null
-        && event.type === 'keyup'
-        && $quantityWantedInput.val() === $quantityWantedInput.data('old-value')
+    if (event !== null &&
+        event !== undefined &&
```

Comment:
If `event` may not be defined, you have to perform this check instead:
```js
typeof error === 'undefined'
```

Otherwise, a `ReferenceError` will be thrown.

Reply:
`event` is always `defined` (event if it can have `undefined` value) because of signature.